beps/bep_0046.rst

```diff
@@ -40,9 +40,19 @@ Publishing content
 ==================
 
 Publishers should issue a mutable ``put`` request when they want to notify
-consumers about an update of a torrent. The value of the payload ``v`` is the 20
-byte infohash of such torrent. Note that there is a 1-to-1 mapping between a
-mutable DHT item and a torrent.
+consumers about an update of a torrent. The value of the payload ``v`` contains
+a bencoded dict with key ``ih`` (info hash) and value the 20 byte infohash of
```

Comment:
remove the bencoded, it just contains a dictionary. if it were bencoded it would be a string.